docs/USING_TYPES.md

```diff
@@ -0,0 +1,39 @@
+# Types
+
+In you use redux logging tool like redux-logger, you can see the action types that React-Redux-Grid produces as you are interacting with the grid. 
+
+You have may other components that may wish to listen to these commands and you can use the React-Redux-Grid types in your own reducers.
+
+#### Grid Types
```

Comment:
Ahh ok... I saw different types within it... there are GirdConstants.js and ActionTypes.js 

GridConstants are already exposed as part of applyGridConfig